src/rt-app_parse_config.c

```diff
@@ -791,6 +789,21 @@ static sched_data_t *parse_sched_data(struct json_object *obj, int def_policy)
 	tmp_data.period = get_int_value_from(obj, "dl-period", TRUE, tmp_data.runtime);
 	tmp_data.deadline = get_int_value_from(obj, "dl-deadline", TRUE, tmp_data.period);
 
+	/* clamping params (-1: no changes ) */
+	tmp_data.boost = get_int_value_from(obj, "boost", TRUE, -1);
```

Comment:
uclamp_max is a better name; it'd be much clearer to keep the 1:1 naming since the user is expected to know they're using uclamp and renaming it to make it look like something else will add confusion

Reply:
Done